src/devdocs2zim/entrypoint.py

```diff
@@ -24,6 +25,7 @@ def zim_defaults() -> ZimConfig:
         description_format="{full_name} docs by DevDocs",
         long_description_format=None,
         tags="devdocs;{slug_without_version}",
+        logo_format=str(DEFAULT_LOGO_PATH.absolute()),
```

Comment:
Why do we need to make it absolute? I mean, execution context will not change, so the path we have here, even if relative, will still resolve the same in the whole program, even if we do not make it absolute.

Reply:
I thought it would be easier to understand in descriptions or errors, but you're right that it's not needed so I removed it.